engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -222,26 +214,21 @@ private void resetStats() {
         statRenderedTriangles = 0;
     }
 
-    private void preRenderUpdate(DefaultRenderingProcess.StereoRenderState stereoRenderState) {
-
-        switch (stereoRenderState) {
-            case MONO:
-                currentRenderingStage = WorldRenderingStage.DEFAULT;
-                isFirstRenderingStageForCurrentFrame = true;
-                break;
-            case OCULUS_LEFT_EYE:
-                currentRenderingStage = WorldRenderingStage.OCULUS_LEFT_EYE;
-                isFirstRenderingStageForCurrentFrame = true;
-                break;
-            case OCULUS_RIGHT_EYE:
-                currentRenderingStage = WorldRenderingStage.OCULUS_RIGHT_EYE;
-                isFirstRenderingStageForCurrentFrame = false;
-                break;
+    private void preRenderUpdate(WorldRenderingStage renderingStage) {
+
+        currentRenderingStage = renderingStage;
+        if (currentRenderingStage == WorldRenderingStage.MONO || currentRenderingStage == WorldRenderingStage.OCULUS_LEFT_EYE) {
+            isFirstRenderingStageForCurrentFrame = true;
+        } else {
+            isFirstRenderingStageForCurrentFrame = false;
         }
 
         // this is done to execute this code block only once per frame
         // instead of once per eye in a stereo setup.
         if (isFirstRenderingStageForCurrentFrame) {
+            tick += secondsSinceLastFrame * 1000;  // Updates the tick variable that animation is based on.
+            smoothedPlayerSunlightValue = TeraMath.lerp(smoothedPlayerSunlightValue, getSunlightValue(), secondsSinceLastFrame);
```

Comment:
Took the plunge and moved these two lines from the WorldRendererLwjgl.update() method to here. I suspect the move is inconsequential apart from these lines not being executed once per tick/eye anymore.